### PR TITLE
Accept a classpath option

### PR DESCRIPTION
**Pull request by w4tson**
Initial support for classpaths. Currently only supports a a single jar.

- Used structopts crate for ease of use in parsing commandline options

Created a noddy jar to test. Foo depends on Bar and adds 1 to 42.

Invoked like 

`cargo run -- --classpath resources/test/jar/foo.jar Foo`

or 

`ocelotter --classpath resources/test/jar/foo.jar Foo` once the binary is fully built

### CODE REVIEW

File: src/options.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    pub classname: Vec<String>
+}
+
+impl Options {
+    pub fn fq_klass_name(&self) -> String {
+        format!("{}.class", self.f_name())
+    }
+    
+    pub fn f_name(&self) -> String {
+        self.classname.get(0).expect("Classname should be specified").into()
```
Comment (kittylyst) [inline src/options.rs:23]: As it stands, this doesn't compile out of the box. I think the call chain here should be get().expect().as_str().into() due to the trait implementations. Do you agree?

**Review by kittylyst — COMMENTED** [review]
I'm still working my way through these changes, but this makes sense.

File: src/options.rs (hunk 1 above)
Comment (w4tson) [reply src/options.rs:23]: Running on 1.36 (which was originally used to develop on at Devoxx) and 1.39 stable using `cargo run -- --classpath resources/test/jar/foo.jar Foo` returns `43`, no compilation errors.

Comment (kittylyst) [reply src/options.rs:23]: Looks like I'm running 1.34 - this type of API shake is one of the things I *don't* like about Rust tbh. I'll rustup update and confirm.

**Pull request merged**